README.md

```diff
@@ -114,7 +114,7 @@ You can pass the following options during the plugin registration:
 await fastify.register(import('@fastify/rate-limit'), {
   global : false, // default true
   max: 3, // default 1000
-  ban: 2, // default null
+  ban: 2, // default Infinity
```

Comment:
This change likely needs a bit more docs than this

Reply:
Yeah probably types should be changed as well… will take a look at it today

Reply:
So wait. Null could mean disabled and 0 could a mean immediatly block. Better than infinity?

Reply:
I would split this into two configs minumum

Reply:
Should he implement it or is this a suggestion?

Reply:
So something like `ban` and `banCanBe0`?

I think just applying the same logic for `ban` when it's defined as a numeric value (including 0) is simpler